File: src/main/java/arcana/data/BlockstateProvider.java
package arcana.data;

import arcana.Arcana;
import net.minecraft.data.DataGenerator;
import net.minecraftforge.client.model.generators.BlockStateProvider;
import net.minecraftforge.common.data.ExistingFileHelper;

import static arcana.common.blocks.ModBlocks.*;

public class BlockstateProvider extends BlockStateProvider {
    public BlockstateProvider(DataGenerator gen, ExistingFileHelper exFileHelper) {
        super(gen, Arcana.id, exFileHelper);
    }

    @Override
    protected void registerStatesAndModels() {
    }
}

File: src/main/kotlin/arcana/data/BlockstateProvider.kt
package arcana.data

import arcana.Arcana
import net.minecraft.data.DataGenerator
import net.minecraftforge.client.model.generators.BlockStateProvider
import net.minecraftforge.common.data.ExistingFileHelper

class BlockstateProvider(gen: DataGenerator, exFileHelper: ExistingFileHelper)
    : BlockStateProvider(gen, Arcana.id, exFileHelper) {
    override fun registerStatesAndModels() {}
}